src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,40 +33,56 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
+    //--------------------------------------------------------------------------
 
-    Application& app_;
-    beast::Journal m_journal;
-    LockType mLock;
+    Application& mApp;
+    beast::Journal mJournal;
```

Comment:
A trailing underscore is clearly better than an `m` prefix, and therefore more correct. Given a choice between adding data members with trailing underscores (and not renaming existing members) versus adding new data members with an `m` prefix and changing existing members to consistently have an `m` prefix, which do you feel is the better choice?